implementations/python/sugar-lift-py-tests/tests/fixtures/with_v2_laws/authority_alias.py

```
@@ -0,0 +1,8 @@
+from provider_authority import DoorB as Q
+class Sugar: pass
```

Ruff (0.15.21)

[error] Multiple statements on one line (colon)

(E701)